### PR TITLE
[PROFTPD] Add directories creation for virtuals users

### PR DESCRIPTION
**Pull request by CiolFr**
Add automatic home directories creation for virtual users

### CODE REVIEW

File: roles/proftpd/tasks/users.yml [hunk 1]
```diff
@@ -5,3 +5,12 @@
     src:  "{{ manala_proftpd_users_template|ternary(manala_proftpd_users_template, 'users/default.j2') }}"
     dest: "{{ manala_proftpd_users_file }}"
     mode: 0600
+
+- name: users > home directories
+  file:
+    path: "{{ item.home }}"
+    owner: manala_proftpd_users_default_uid
```
Comment (nervo) [inline roles/proftpd/tasks/users.yml]: should be `user.uid|default(manala_proftpd_users_default_uid)` i guess

File: roles/proftpd/defaults/main.yml [hunk 2]
```diff
@@ -15,3 +15,5 @@ manala_proftpd_configs:           []
 manala_proftpd_users_file:        /etc/ftpd.passwd
 manala_proftpd_users_template:    ~
 manala_proftpd_users:             []
+manala_proftpd_users_default_uid: 108
```
Comment (nervo) [inline roles/proftpd/defaults/main.yml]: Could you convert it to
```
manala_proftpd_users_defaults:
    uid: 108
    gid: 100
```
to keen consistance across roles :)

**Comment by nervo** [comment]
Tests failed: https://travis-ci.org/github/manala/ansible-roles/jobs/667795948?utm_medium=notification&utm_source=github_status

Could you have a look?

File: roles/proftpd/tasks/users.yml (hunk 1 above)
Comment (CiolFr) [reply roles/proftpd/tasks/users.yml]: Done ! thx

File: roles/proftpd/defaults/main.yml (hunk 2 above)
Comment (CiolFr) [reply roles/proftpd/defaults/main.yml]: Done ! thx

**Comment by CiolFr** [comment]
@nervo This build seems to have a timeout issue. But I don't think this a related to my commit. Is there anyway to rerun it ?

**Comment by nervo** [comment]
@CiolFr Yep, this was just a timeout :)
Could you just add an entry in the changelog ?

**Comment by CiolFr** [comment]
@nervo  :( still the problem

**Comment by nervo** [comment]
@CiolFr don't worry, it's not your fault :)

**Comment by greedybro** [comment]
@CiolFr i just release manala_proftpd 1.0.8 with your changes ! Thx for contributing

**Pull request merged**